{
	"name": "@offirmo-private/react-error-boundary",
	"description": "Generic error boundary for React, with a fallback UI and logging",
	"version": "0.0.1",
	"author": "Offirmo <offirmo.net@gmail.com> (https://www.offirmo.net/)",
	"license": "Unlicense",
	"private": true,
	"sideEffects": false,
	"type": "module",
	"exports": {
		".": {
			"import": "./module/index.js"
		}
	},
	"source": "module/index.js",
	"peerDependencies": {
		"@offirmo-private/soft-execution-context": "*",
		"react": "^19"
	},
	"dependencies": {
		"@offirmo-private/async-utils": "*",
		"memoize-one": "^6",
		"prop-types": "^15",
		"tiny-invariant": "^1"
	},
	"scripts": {
		"clean": "monorepo-script--clean-package …dist …cache",
		"start:parcel": "parcel --out-dir .parcel --no-autoinstall doc/demo/index.html",
		"demo": "run-s clean start:parcel"
	},
	"devDependencies": {
		"@offirmo-private/monorepo-scripts": "*",
		"@offirmo-private/soft-execution-context": "*",
		"@offirmo/unit-test-toolbox": "^9",
		"npm-run-all": "^4",
		"react": "^19",
		"react-dom": "^19"
	}
}
